Vishnu/Assignment3.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+</head>
+<body>
+<form action="" method="post">
+<h1>Register</h1>
+<p>
+    First Name <input type="text" name="FirstName" required>
```

Comment:
```suggestion
    First Name <input type="text" name="FirstName" required autocomplete="given-name">
```

This an example of how to use `autocomplete` , use this same format for the other fields marked as `autocomplete` missing